Dockerfile

```diff
@@ -5,41 +5,60 @@ FROM base AS deps
 # Check https://github.com/nodejs/docker-node/tree/b4117f9333da4138b03a546ec926ef50a31506c3#nodealpine to understand why libc6-compat might be needed.
 RUN apk add --no-cache libc6-compat
 WORKDIR /app
-COPY package.json package-lock.json ./
+
+# Install dependencies based on the preferred package manager
+COPY package.json package-lock.json*  ./
 COPY npm npm
-RUN npm install --legacy-peer-deps
+RUN npm ci
+
+
 
 # Rebuild the source code only when needed
 FROM base AS builder
 WORKDIR /app
-COPY . .
+
 COPY --from=deps /app/npm ./npm
 COPY --from=deps /app/node_modules ./node_modules
-RUN npm run build && npm install --legacy-peer-deps --production --ignore-scripts --prefer-offline
+COPY . .
+
+
+# Next.js collects completely anonymous telemetry data about general usage.
+# Learn more here: https://nextjs.org/telemetry
+# Uncomment the following line in case you want to disable telemetry during the build.
+ENV NEXT_TELEMETRY_DISABLED 1
+
+RUN npm install sharp
```

Comment:
We have already included sharp in the package.json file, is this needed?

Reply:
this is the error i'm getting without the above command.
`
 Build error occurred
[Error: ENOENT: no such file or directory, copyfile '/app/node_modules/sharp/build/Release/sharp-linux-x64.node' -> '/app/.next/standalone/node_modules/sharp/build/Release/sharp-linux-x64.node'] {
  errno: -2,
  code: 'ENOENT',
  syscall: 'copyfile',
  path: '/app/node_modules/sharp/build/Release/sharp-linux-x64.node',
  dest: '/app/.next/standalone/node_modules/sharp/build/Release/sharp-linux-x64.node'
}
`

Reply:
Checking

Reply:
I suspect this is happening because we don't have `.next` in .dockerignore file. So it's likely that `.next` gets transferred to the docker layer messing up the binary.

Reply:
@G3root  You could try with below dockerignore
```ignore
Dockerfile
.dockerignore
**/node_modules
npm-debug.log
README.md
.next
.git
.github
_dev
.vscode
swagger
uffizzi
```

Reply:
i'll try it today